data/migrations/20210713225521_updated-table.js

```diff
@@ -0,0 +1,35 @@
+exports.up = function (knex) {
+  return knex.schema
+    .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
```

Comment:
is this extension necessary? I don't see 'uuid' as a data type for any of the table columns